ods_ci/tests/Tests/100__deploy/100__installation/111__trustyai.robot

```diff
@@ -11,29 +13,31 @@ ${TRUSTYAI_NS}=    ${APPLICATIONS_NAMESPACE}
 Verify TrustyAI Operator Installation
     [Documentation]    Verifies that the TrustyAI operator has been
     ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
-    [Tags]    Smoke
-    ...       Tier1    ODS-2481
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Deployment
-    Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI ReplicaSets Info
-    Wait Until Keyword Succeeds    10 times  5s    Verify TrustyAI Container Names
+    [Tags]    Tier1
+    ...       ODS-2481
+    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  1 min  10 sec  Verify TrustyAI Operator Deployment
```

Comment:
(Optional improvement): Instead of adding ` Run Keyword And Continue On Failure` on all lines you can add `robot:continue-on-failure` or `robot:recursive-continue-on-failure` in the  Tags section

https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#enabling-continue-on-failure-using-tags

Reply:
Done, Thanks! Learned something new.